js/controllers/settings.js

```diff
@@ -60,6 +60,28 @@ SettingsController.prototype.show_ = function(key, value) {
   }
 };
 
+/**
+ * Activates a switch Material Component element in the UI.
+ * @param {string} key The unique section of the id of the switch element
+ *     (after the 'setting-' prefix).
+ */
+SettingsController.prototype.checkSwitch_ = function(key) {
+  document.getElementById('setting-' + key).setAttribute('checked', '');
```

Comment:
Can you use https://developer.mozilla.org/en-US/docs/Web/API/Element/toggleAttribute? I think there's also a `toggleClass` on classlist too.

Reply:
Currently this code is used to do more than just toggle, sometimes we are setting to false when it's already false (e.g. in initial setup of all the settings switches). Do you want me to change the setup code to instead check the value and conditionally toggle if the initial value is supposed to be true?

Reply:
oh actually I can just use the second argument of the toggle function to state whether the switch should be turned on or off.